src/__tests__/__snapshots__/postcss.js.md

```diff
@@ -124,16 +124,6 @@ Generated by [AVA](https://ava.li).
         |  ^␊
     `
 
-## bad whitespace instead of namespace token
```

Comment:
Strange, WIP on this

Reply:
We remove this test and forget update snapshot, strange what old version of `ava` doesn't exit with 1 code

Reply:
Are you saying it is okay to remove this test?

Reply:
@jonathantneal yep, we remove this tests in past, but doesn't update snapshot (i don't know why), `jest` exit with non zero code when `snapshot` contains extra snapshots but `ava` doesn't do this in old versions